lib/matplotlib/text.py

```diff
@@ -232,6 +234,7 @@ def __init__(self,
             linespacing = 1.2   # Maybe use rcParam later.
         self._linespacing = linespacing
         self.set_rotation_mode(rotation_mode)
+        self._trans_rotate = trans_rotate
```

Comment:
should set to `None` and then use the setter to get the validation the `__init__` input as well.